bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/it/CloudBigtableBeamIT.java

```diff
@@ -142,8 +147,11 @@ public void testWriteToBigtable() throws IOException {
     DataflowPipelineOptions options = PipelineOptionsFactory.as(DataflowPipelineOptions.class);
     properties.applyTo(options);
     options.setAppName("testWriteToBigtable-" + System.currentTimeMillis());
+    options.setExperiments(Collections.singletonList("disable_runner_v2"));
```

Comment:
runner v2 (which is used by default) doesn't have the support for retrieving lineage metrics yet.

Reply:
We'll wait for the support to be added before moving forward with these changes. Confirmed that changes work fine otherwise.